### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by phargob** [OSRSB/OsrsBot#235]
**BEFORE REPORTING**
Ensure RuneLite is up to date in IDE and you are using the latest version of the master branch

**Describe the bug**
walking.setRun() is not working

**Expected behavior**
Should be able to toggle running/walking.

**Additional context**
I tracked down the issue to:
```methods.interfaces.getComponent(GlobalWidgetInfo.MINIMAP_RUN_ORB)```
we need to update MINIMAP_RUN_ORB I guess.

For instance this works:
```ctx.interfaces.getComponent(WidgetIndices.Minimap.GROUP_INDEX, 27).doClick()```

**Comment by GigiaJ** [comment]
Yeah. A lot of the Widget changes Bia made are going to break over time....
